app/script/conversation/ConversationRepository.js

```diff
@@ -3534,6 +3401,7 @@ z.conversation.ConversationRepository = class ConversationRepository {
           conversationEntity.prepend_messages(messageEntities);
         } else {
           conversationEntity.add_messages(messageEntities);
+          messageEntities.forEach(this.timedMessageHandler.addTimedMessage);
```

Comment:
You also need to do this in the prepend case. Take it out of the if block. Otherwise message you reveal when scrolling up are not restarted.